### ISSUE TITLE
Suggestions on preventing collapse?

### ISSUE THREAD

**Issue by NoTody** [facebookresearch/vjepa2#109]
Hi, I'm training my model with vjepa objective on own datasets. I observed that the collapse happens after long time annealing. Do you have any suggestions to prevent this kind of loss collapse (e.g. what hyperparameters I should tweak?) The attachment is my training loss plot.

<img width="999" height="396" alt="Image" src="https://github.com/user-attachments/assets/6555994e-73ed-4eb4-acba-a11311330e68" />
